packages/core-app-api/api-report.md

```diff
@@ -259,6 +262,7 @@ export type AuthApiCreateOptions = {
 export type BackstageApp = {
   getPlugins(): BackstagePlugin[];
   getSystemIcon(key: string): IconComponent | undefined;
+  createRoot(element: JSX.Element): ComponentType<{}>;
```

Comment:
Very much up for suggestions on the naming here. Some other stuff I tried was `.wrap()` and `.createComponent`, but I felt `.createRoot` made the most sense in the end.

Reply:
I like it.

Hm I thought the getters below would have been visibly deprecated in this file too, maybe I'm just misremembering.